Pipfile

```diff
@@ -56,6 +56,23 @@ dictdiffer = "*"
 setuptools = "*"
 markupsafe = "==2.0.1"
 imagesize = "*"
+cryptography = "*"
+# linter-related
+typed-ast = "*"
+types-dateparser="*"
+types-pyyaml="*"
+types-requests="==2.26.3"# This is an open issue https://github.com/python/typeshed/issues/6893
```

Comment:
Fixed. you can unlock it, just add urlib3 stubs to the installation and release this version.